### PR TITLE
Cubrid enhancement

### PR DESCRIPTION
**Pull request by saundefined**
Mostly changed `return values` section.
Some values wrapped with `literal` tag.

### CODE REVIEW

File: reference/cubrid/functions/cubrid-get-charset.xml [hunk 1]
```diff
@@ -35,10 +35,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   A string that represents the CUBRID connection charset; on success.
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-get-charset.xml:38]: The semicolon makes the sentence; weird. Can you fix it, please?

File: reference/cubrid/functions/cubrid-get-client-info.xml [hunk 2]
```diff
@@ -28,10 +28,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   A string that represents the client library version; on success.
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-get-client-info.xml:31]: ditto

File: reference/cubrid/functions/cubrid-get-db-parameter.xml [hunk 3]
```diff
@@ -14,7 +14,7 @@
    <methodparam><type>resource</type><parameter>conn_identifier</parameter></methodparam>  
   </methodsynopsis>
   <para>
-   This function returns the CUBRID database parameters or it returns FALSE on
+   This function returns the CUBRID database parameters or it returns &false; on
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-get-db-parameter.xml:17]: Why not use `&return.falseforfailure;` here also?

File: reference/cubrid/functions/cubrid-get-server-info.xml [hunk 4]
```diff
@@ -33,10 +33,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   A string that represents the CUBRID server version; on success.
-  </para>
-   <para>
-    &false; on failure.
+   A string that represents the CUBRID server version; on success,&return.falseforfailure;.
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-get-server-info.xml:36]: Ditto

File: reference/cubrid/functions/cubrid-lob-close.xml [hunk 5]
```diff
@@ -25,7 +25,7 @@
   <variablelist>
     <varlistentry>
   <term><parameter>lob_identifier_array</parameter></term>
-  <listitem><para>LOB identifier array return from cubrid_lob_get.</para></listitem>
+  <listitem><para>LOB identifier array return from <function>cubrid_lob_get</function>.</para></listitem>
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-lob-close.xml:28]: `returned` maybe?

File: reference/cubrid/functions/cubrid-lob2-read.xml [hunk 6]
```diff
@@ -49,7 +49,7 @@
     &false; when there is no more data.
   </para>
   <para>
-    NULL on failure.
+    &null; on failure.
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-lob2-read.xml:52]: Not sure we need 3 paragraphs for that.

File: reference/cubrid/functions/cubrid-move-cursor.xml [hunk 7]
```diff
@@ -20,18 +20,18 @@
     current cursor location of <parameter>req_identifier</parameter> by the
     value set in the <parameter>offset</parameter> argument, to the direction
     set in the <parameter>origin</parameter> argument. To set the
-    <parameter>origin</parameter> argument, you can use CUBRID_CURSOR_FIRST
-    for the first part of the result, CUBRID_CURSOR_CURRENT for the current
-    location of the result, or CUBRID_CURSOR_LAST for the last part of the
+   <parameter>origin</parameter> argument, you can use <literal>CUBRID_CURSOR_FIRST</literal>
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-move-cursor.xml:23]: What happened to indentation here?

File: reference/cubrid/functions/cubrid-num-rows.xml [hunk 8]
```diff
@@ -15,10 +15,10 @@
...
   <para>
     The <function>cubrid_num_rows</function> function is used to get the
-    number of rows from the query result.  You can use it only when the query
-    executed is a select statement.  When you want to know such value for
-    INSERT, UPDATE, or DELETE query, you have to use the
-    <function>cubrid_affected_rows</function> function.
+    number of rows from the query result. You can use it only when the query
+    executed is a select statement. When you want to know such value for
+    <literal>INSERT</literal>, <literal>UPDATE</literal>, or <literal>DELETE</literal> query, 
+    you have to use the <function>cubrid_affected_rows</function> function.
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-num-rows.xml:21]: If we are touching this paragraph at all, the please rewrite it in a non-personal style, i.e. avoid `you`

File: reference/cubrid/functions/cubrid-schema.xml [hunk 9]
```diff
@@ -21,8 +21,7 @@
     requested schema information from database. You have to designate
     <parameter>class_name</parameter>, if you want to get information on
     certain class, <parameter>attr_name</parameter>, if you want to get
-    information on certain attribute (can be used only with CUBRID_
-    SCH_ATTR_PRIVILEGE).
+   information on certain attribute (can be used only with <literal>CUBRID_SCH_ATTR_PRIVILEGE</literal>).
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-schema.xml:24]: I'd rewrite this paragraph too, avoid `you` and the phrase "get information on certain attribute" doesn't sound very grammatically correct to me.

**Comment by saundefined** [comment]
@kamil-tekiela, thank you!

File: reference/cubrid/functions/cubrid-move-cursor.xml [hunk 10]
```diff
@@ -20,18 +20,18 @@
     current cursor location of <parameter>req_identifier</parameter> by the
     value set in the <parameter>offset</parameter> argument, to the direction
     set in the <parameter>origin</parameter> argument. To set the
-    <parameter>origin</parameter> argument, you can use CUBRID_CURSOR_FIRST
-    for the first part of the result, CUBRID_CURSOR_CURRENT for the current
+    <parameter>origin</parameter> argument, you can use <literal>CUBRID_CURSOR_FIRST</literal>
+    for the first part of the result, <literal>CUBRID_CURSOR_CURRENT</literal> for the current
     location of the result, or CUBRID_CURSOR_LAST for the last part of the
```
Comment (kamil-tekiela) [inline reference/cubrid/functions/cubrid-move-cursor.xml:25]: Why did you remove `<literal>` around this constant?

Comment (cmb69) [reply reference/cubrid/functions/cubrid-move-cursor.xml:25]: That should probably be `<constant>` throughout the cubrid book.

Comment (saundefined) [reply reference/cubrid/functions/cubrid-move-cursor.xml]: @cmb, thanks! fixed that and the rest

**Review by kamil-tekiela — CHANGES_REQUESTED** [review]

**Review by kamil-tekiela — APPROVED** [review]

**Review by cmb69 — APPROVED** [review]

**Pull request merged**